src/scenes/start-scene/start-scene.ts

```diff
@@ -68,7 +67,8 @@ export class StartScene {
       height: this.canavsElement.height,
       onLoad: () => {
         this.riveMonster.play(RiveMonsterComponent.Animations.IDLE); // Start with the "Idle" animation
-      }
+      },
+      gameCanvas: this.canvas
```

Comment:
Start scene doesn't need the hitbox. Make this optional.

Reply:
@bernhardccodev thanks i applied an adjustment for this.